test/api/common/HttpClient.test.ts

```diff
@@ -0,0 +1,87 @@
+import type { EnvelopingTypes } from '@rsksmart/rif-relay-contracts/dist/typechain-types/contracts/RelayHub';
+import { expect, use } from 'chai';
+import { createSandbox } from 'sinon';
+import sinonChai from 'sinon-chai';
+import { HttpClient, HttpWrapper } from '../../../src/api/common';
```

Comment:
should really sort out absolute paths at some point to avoid double-dot hell

Reply:
shouldn't we configure `paths` in `tsconfig.json`?

Reply:
Let's handle that in a separate task tho